clients/banking/src/components/NewMembershipWizard.tsx

```diff
@@ -468,9 +472,13 @@ export const NewMembershipWizard = ({
                 .otherwise(data => {
                   match(__env.ACCOUNT_MEMBERSHIP_INVITATION_MODE)
                     .with("EMAIL", () => {
-                      sendInvitation({ editingAccountMembershipId: data.id });
+                      sendInvitation({
+                        editingAccountMembershipId: data.id,
+                        preferredEmailLanguage,
```

Comment:
the preferred language is missing from the `addAccountMembership` call.

also, it seems like the languages aren't in sync as the API is using `AccountLanguage`, which doesn't have finnish.